web/themes/custom/nicsdru_dept_theme/templates/content/node--search-metadata.html.twig

```diff
@@ -33,3 +28,4 @@
     {% endif -%}
   </div>
 </a>
+{{ content | without('field_summary', 'field_teaser', 'field_domain_source', 'field_published_date') }}
```

Comment:
Looks weird, but you need to do this with more custom templates as it ensures the correct cache tags and set when concluding the rendering pipeline otherwise the render process isn't always sure whether everything's been rendered and may not always correctly finalise the render pipeline

Reply:
Basically, if you custom render values but your display mode is configured to render these values then you have to either render the via {{ content.thingy }} or finalise with a final {{ content | without('field_names') }} to get the desired result